application/src/main/java/org/thingsboard/server/service/state/DeviceStateService.java

```diff
@@ -27,11 +27,13 @@
  */
 public interface DeviceStateService extends ApplicationListener<PartitionChangeEvent> {
 
-    void onDeviceConnect(TenantId tenantId, DeviceId deviceId);
+    void onDeviceConnect(TenantId tenantId, DeviceId deviceId, long lastConnectTime);
```

Comment:
default void onDeviceConnect(TenantId tenantId, DeviceId deviceId, long lastConnectTime){
   onDeviceConnect(tenantId, deviceId, System.currentTimeMillis());
};

Reply:
Fixed in https://github.com/thingsboard/thingsboard/pull/9030/commits/1822703730ee2bace9aed2ebc907df8e79dea4c2